kairon/nlu/classifiers/llm.py

```
@@ -251,11 +250,11 @@ def load(
                 file_name = cls.__name__
 
                 vector_file = os.path.join(model_path, file_name + "_vector.db")
-                data_file = os.path.join(model_path, file_name + "_data.pkl")
+                data_file = os.path.join(model_path, file_name + "_data.json")
 
                 if os.path.exists(vector_file):
                     vector = faiss.read_index(vector_file)
-                    data = io_utils.json_unpickle(data_file)
+                    data = json.load(open(data_file, "r"))
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)